
---- START ---- Test/stack_alloc3.c
Test/stack_alloc3.c:10: param decl x2
Test/stack_alloc3.c:12: param use x2
---- STOP ----

--- Test/stack_alloc3.c
+++ Test/stack_alloc3.c
@@ -14,2 +14,6 @@
-  int y2;
-  int *y2_p = &y2;              // and addr taken: yes
+  int 
+// xform this dname
+y2;
+  int *y2_p = &
+// xform this use
+y2;              // and addr taken: yes
@@ -18,2 +18,6 @@
-  struct Foo foo1;              // function/auto: stack
-  struct Foo *foo1_p = &foo1;   // and addr taken: yes
+  struct Foo 
+// xform this dname
+foo1;              // function/auto: stack
+  struct Foo *foo1_p = &
+// xform this use
+foo1;   // and addr taken: yes
@@ -21,2 +21,6 @@
-  struct Foo foo3;              // function/auto: stack
-  int *foo3_q_p = &(foo3.q);    // and addr taken: yes
+  struct Foo 
+// xform this dname
+foo3;              // function/auto: stack
+  int *foo3_q_p = &(
+// xform this use
+foo3.q);    // and addr taken: yes
